README.md

```diff
@@ -25,34 +37,41 @@ Example installation:
 
 `spack install dla-future ^intel-mkl`
 
-Notice that, for the package to work correctly, the HPX option `max_cpu_count` must be set accordingly to the platform,
-as it represents the maximum number of OS-threads.
+Notice that, for the package to work correctly, the HPX option `max_cpu_count` must be set accordingly to the architecture, as it represents the maximum number of OS-threads.
 
 `spack install dla-future ^intel-mkl ^hpx max_cpu_count=256`
 
-## How to use the library
+#### Build the old good way
 
-Using DLAF in a CMake project is extremely easy!
+You can build all the dependencies by yourself, but you have to ensure that:
+- BLAS/LAPACK implementation is not multithreaded
+- HPX: `HPX_WITH_NETWORKING=none` + `HPX_WITH_MAX_CPU_COUNT=n` (according to number of cores in the architecture, suggested the next closest power of 2)
+
+TODO configure cmake script for DLAF (with/without MKL)
 
-Let's use the variable `$DLAF_ROOT` for referring to the install path of DLAF.
+### Link your program/library with DLAF
+
+Using DLAF in a CMake project is extremely easy!
 
 Configure your project with:
 
 ```bash
-cmake -DDLAF_DIR="$DLAF_ROOT/lib/cmake" ..
+# DLAF_INSTALL_PREFIX must be set to where you installed DLAF (i.e. CMAKE_INSTALL_PREFIX)
+cmake -DDLAF_DIR="$DLAF_INSTALL_PREFIX/lib/cmake" ..
```

Comment:
or to add `$DLAF_INSTALL_PREFIX` to `CMAKE_PREFIX_PATH`

Reply:
I'm not used to changing `CMAKE_PREFIX_PATH`, because I feel like changing a "system" thing. What would you want to add exactly. Feel free to propose an actual suggestion for the documentation 👍

Reply:
I think changing `CMAKE_PREFIX_PATH` is a perfectly fine way to help CMake find packages. It has the benefit of users not having to know in which subdirectory the DLAF CMake config file is (you're also using `GNUInstallDirs` which means that the config file can be in `lib/cmake` or `lib64/cmake` depending on the system.

Reply:
Good point. I've just made it the first choice and re-arranged a bit the information of this section.